README.md

```diff
@@ -46,154 +44,3 @@ To disable it you can your settings in chrome: chrome://flags/#allow-insecure-lo
 yarn dev:electron
 ```
 
-#### Build Releases
-
-```bash
-# builds the production server app
-yarn build
-```
-
-```bash
-# builds the downloadable version of the site
-yarn build:downloadable
-```
-
-```bash
-# builds the electron apps
-yarn build:electron
-
-# builds only one OS's electron app
-yarn build:electron:(osx|linux|windows)
-```
-
-All of these builds are output to a folder in `dist/`.
-
-#### Unit Tests:
-
-```bash
-# run unit tests with Jest
-yarn test
-```
-
-#### Integration Tests:
-
-```bash
-# run integration tests with Jest
-yarn test:int
-```
-
-#### Address Derivation Checker:
```

Comment:
Open question if we really want to remove the derivation checker?

Reply:
Yes!

Well

Remove it from this repo.

The only thing in this repo should be a super simple function that outputs a testfile with like 10k private keys and their derived addresses using the methods and libs we use in our website/app

We can then use that file as input in another repo.

Reply:
Honestly most issues occur with a 1/128 probability so 10k is serious overkill. 2k would be super safe, 1k would be safe. 200 would probablistically get 1-2 wrong addresses and be safe.

Reply:
Also, realistically we should run this test like once a quarter or whenever libs are updated or derivation logic is changed (which should be at least that rare)

Reply:
I created a card to move the test into another repo
https://app.clubhouse.io/mycrypto/story/3489/create-test-for-derivation-checker